tests/test_automl.py

```diff
@@ -15,8 +15,12 @@ class TestAutoMl(unittest.TestCase):
     def test_version(self):
         self.assertIsNotNone(automl.auto_ml_client._GAPIC_LIBRARY_VERSION)
         version_parts = automl.auto_ml_client._GAPIC_LIBRARY_VERSION.split('.')
-        version = float('.'.join(version_parts[0:2]));
-        self.assertGreaterEqual(version, 0.5);
+        self.assertEqual(3, len(version_parts))
```

Comment:
Shouldn't this be 2?

Reply:
This package is using `semver` (i.e. `major.minor.patch`).

Reply:
Maybe we should a package?
https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python/21065570

Reply:
There is a `semver` package on Pypi but I don't think it is worth adding to our base image given this check is very simple. That's the only semver check we are doing in our test.

If we ever need to do another one, we could simply add a function to the `tests/common.py` file. I don't think adding yet another dependency is worth it.

Reply:
Looking at the link you posted, we can simply use the `packaging.version` package that is part of Python (no need for the separate `semver` package).

Reply:
Done, I am now using `packaging.version`.